src/ACEtk/details/verify/src/size.hpp

```diff
@@ -0,0 +1,10 @@
+template< typename R1, typename R2 >
+inline void equalSize( R1&& a1, R2&& a2 ){
```

Comment:
`inline` is implicit on function templates. Please remove the keyword.

Here, we ought not mutate `R1` and `R2`. Please accept arguments by const reference.